settings.py

```diff
@@ -11,7 +11,7 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        "NAME": os.path.join(BASE_DIR, "db.sqlite"),
```

Comment:
Consider using `os.path.join(BASE_DIR, 'db.sqlite3')` instead of `db.sqlite` to follow the convention of naming SQLite files with a `.sqlite3` extension.